hp/elitebook/830/g6/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, lib, ... }:
+{
+  imports = [
+    ../../../../common/cpu/intel
+    ../../../../common/pc/laptop
+    ../../../../common/pc/laptop/acpi_call.nix
+    ../../../../common/pc/laptop/ssd
+  ];
+
+  # is there any redistributable firmware for this device?
```

Comment:
Can you figure this out before enabling it by default?

Reply:
Pretty sure the the intel wifi ship needs it (Intel Wi-Fi 6 AX200)